api/src/pcapi/routes/adage_iframe/serialization/adage_authentication.py

```diff
@@ -30,6 +30,15 @@ class AuthenticatedInformation(BaseModel):
     lon: float | None
 
 
+class EducationalInstitutionProgramModel(BaseModel):
+    name: str
+    label: str | None
+    description: str | None
+
+    class Config:
+        orm_mode = True
```

Comment:
pour des raisons de cohérence et pour aider les copier/coller:
```suggestion
        orm_mode = True
        alias_generator = to_camel
        allow_population_by_field_name = True
```